### PR TITLE
Add option to streamspraydf to allow for different potentials for orbit integration and tidal-radius calculation

### PR DESCRIPTION
**Pull request by webbjj**
added option to streamspray that allows for different potential to be used for tidal radius calculation

### CODE REVIEW

File: streamtools/df/streamspraydf.py [hunk 1]
```diff
@@ -35,7 +35,11 @@ def __init__(self,progenitor_mass,progenitor=None,pot=None,
            meankvec= (Fardal+2015-ish defaults) 
            
            sigkvec= (Fardal+2015-ish defaults) 
+
+           pot = (None) potential for integrating orbits
            
+           rtpot = (None) potential for calculating tidal radius and vircular velocity
```
Comment (jobovy) [inline streamtools/df/streamspraydf.py]: vircular -> circular

I would also write
"rtpot = (pot) potential for calculating tidal radius and vircular velocity"
to make it clear that the default is pot

File: streamtools/df/streamspraydf.py [hunk 2]
```diff
@@ -60,6 +64,12 @@ def __init__(self,progenitor_mass,progenitor=None,pot=None,
         if pot is None: #pragma: no cover
             raise IOError("pot= must be set")
         self._pot= flatten_potential(pot)
+
+        if rtpot is None:
+            self._rtpot=self._pot
+        else:
+            self._rtpot=flatten_potential(rtpot)
+
```
Comment (jobovy) [inline streamtools/df/streamspraydf.py]: Could also do
```
self._rtpot= self._pot if rtpot is None else flatten_potential(rtpot)
```

**Review by jobovy — CHANGES_REQUESTED** [review]
Looks good! Some very minor comments

**Comment by webbjj** [comment]
Changes made. I think I may have accidentally accepted the pull request though. I meant to just update the file. My bad.

**Comment by jobovy** [comment]
🎉🎉🎉

**Pull request merged**